src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -213,9 +213,43 @@ public void UpdateToWorkloadSetVersionWithManifestsNotAvailable()
             GetWorkloadVersion().Should().Be(workloadVersionBeforeUpdate);
         }
 
-        string GetWorkloadVersion()
+        [Fact]
+        public void UpdateWorkloadSetViaGlobalJson()
+        {
+            InstallSdk();
+
+            var versionToUpdateTo = "8.0.300-preview.0.24217.2";
+            var directory = "C:\\SdkTesting";
+
+            string originalVersion = GetWorkloadVersion();
+
+            var rollback = GetRollback(directory);
+
+            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo""}}}").Execute().Should().Pass();
```

Comment:
This needed duplication:
```suggestion
            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo""}}}}}").Execute().Should().Pass();
```